### ISSUE TITLE
transfer projects from old digital logic sim app

### ISSUE THREAD

**Issue by satr14washere** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#86]
I have a lot of projects on the old app but I want to migrate to this app since it has more features but I also don't wanna redo my projects

is there a way to transfer my projects?

**Comment by satr14washere** [comment]
found a solution for those wondering: https://docs.unity3d.com/ScriptReference/Application-persistentDataPath.html

that site contains the save files location

**Comment by satr14washere** [comment]
ok it doesnt work lol

**Comment by lahavg** [comment]
I also have the same problem, I transferred the folder with the name of the file I wanted to the new application and it does see the project but it does not load the logged components

**Comment by VitoBarra** [comment]
Which "old" version are you referring to?
the old-original one or the newer version of the original DLS ?

**Comment by satr14washere** [comment]
> Which "old" version are you referring to?
> the old-original one or the newer version of the original DLS ?

newer original for me

**Comment by satr14washere** [comment]
the issue i see is that the save files uses diffrent formats so you might need to make an app for this

**Comment by VitoBarra** [comment]
Yeah, I'm aware of the issue. I started working on it in pull request #79, but I ran into some problems with the mapping between the built-in chip in the original version and the built-ins in this version. I'm unable to work on it at the moment, so if anyone wants to take it up, they can start from that PR. Perhaps I'll return to work on it in the future.
